docs/man/images/Images_info.adoc

```diff
@@ -0,0 +1,54 @@
+# Images in man pages
+
+Images in man pages are not recommended. It's better to use a ASCII representation (ASCII art) if possible.
+
+Images can be added in the man pages for representation as HTML and PDF
+The images have to be in *.ps-format and should be placed only in this folder.
+They can be included using `.PSPIC -L ../man/images/image.ps`.
+The PNG needed for the HTML view is automatically created on generating LinuxCNC_Manual_Pages.pdf
```

Comment:
Why is this section here, with a recipe on how to do it, if it is not recommended?

Reply:
It was meant as additional information, but if it's confusing, I'll remove it.